### ISSUE TITLE
"Accessibility -> Missing the link to "Skip to main content on A&R website

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#265]
## Description
**How to replicate:**
* Navigate to the A&R website on the Test environment
* Using the keyboard try to navigate through the site

**Issue:**
* The site is missing the option to “Skip to the main content” when navigating using the keyboard

## Linked issues
relates to
-[x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
